spec/unit/run_ecs_tasks_spec.rb

```diff
@@ -48,21 +50,14 @@
         cluster: 'blue',
         task_definition: shipitron_task,
         count: 1,
-        started_by: 'shipitron'
+        started_by: Shipitron::Client::STARTED_BY
       )
     )
   end
 
-  it 'runs the task for the green cluster' do
+  it 'does not run the task for the green cluster' do
```

Comment:
Can you explain this change?

Reply:
We changed this quite a while back but I never updated the spec. The idea is we run the shipitron build task once, but update the services on all configured clusters.

Reply:
Also the configured clusters are discovered via an SRV request.